docs/grisp_connect_api.md

```diff
@@ -30,8 +30,28 @@ We use [jsonrpc](https://www.jsonrpc.org) 2.0 between frontend and backend.
 
 | Error Content                                       | When it Happens                  |
 | ----------------------------------------------------| -------------------------------- |
-| `{code: -10, message: "grisp_updater unavailable"}` | Grisp updater app is not running |
-| `{code: -11, message: "already updating "}`         | An update is already happening   |
+| `{code: -10, message: "grisp_updater_unavailable"}` | Grisp updater app is not running |
+| `{code: -11, message: "already_updating"}`         | An update is already happening   |
+| `{code: -12, message: "boot_system_not_validated"}` | The board rebooted after an update and needs validation |
+
+</p>
+</details>
+
+<details><summary><i>Post - Validate an update</i></summary>
```

Comment:
Can we add a description to it?

Reply:
@sylane can you check/correct my description? not really sure is best worded